### PR TITLE
Add track_features to old python_abi packages

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
This is ready for review

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -800,6 +800,11 @@ def _gen_new_index(repodata, subdir):
                     record["track_features"] = _extract_track_feature(
                         record, feat)
 
+        # add track_features to old python_abi pypy packages
+        if (record_name == 'python_abi' and 'pypy' in record['build'] and
+                "track_features" not in record):
+            record["track_features"] = "pypy"
```
Comment (beckermr) [inline recipe/gen_patch_json.py:806]: Do we want to version this feature with the pypy version?

Comment (isuruf) [reply recipe/gen_patch_json.py:806]: No, we want to match this with the `track_features` in python.

**Comment by wolfv** [comment]
Thanks a lot!

**Pull request merged**